Release/src/http/client/http_client_asio.cpp

```diff
@@ -1101,6 +1099,7 @@ class asio_context final : public request_context, public std::enable_shared_fro
     {
         if (!ec)
         {
+            m_timer.start_communicating();
```

Comment:
As stated above, if a TLS handshake was required, this will now eventually send the request. As before, `m_timer.start_communicating()` switches the `timeout_timer` over to the "global timeout", just before sending the application payload data.